csharp/test/Ice/ami/TestI.cs

```diff
@@ -8,7 +8,10 @@ namespace ZeroC.Ice.Test.AMI
 {
     public class TestIntf : ITestIntf
     {
-        private readonly object _mutex = new object();
+        private readonly object _mutex = new ();
```

Comment:
I updated tests this morning to use `new()` no space to avoid warnings with the latest StyleCop analyzers